app/experimenter/experiments/forms.py

```diff
@@ -1277,10 +1277,47 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean(self):
+        if "other_normandy_ids" in self.cleaned_data:
+
+            for number in self.cleaned_data["other_normandy_ids"]:
+                if number == self.cleaned_data["normandy_id"]:
+                    raise forms.ValidationError(
+                        "Duplicate IDs are not accepted."
+                    )
+
+        return self.cleaned_data
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
+            ids = self.cleaned_data["other_normandy_ids"].split(",")
+            for item in ids:
+                if not item.strip().isdigit():
+                    raise forms.ValidationError(
+                        "IDs must be numbers separated by commas."
+                    )
+
+                other_ids_list.append(int(item))
```

Comment:
So, in THEORY, something could pass `item.strip().isdigit()` but still fail `int(item)` and then this would crash.  If you're going to do `int(item)` anyways (because that's how the db expects it) then you could just let that be your actual test and then catch the `ValueError` and let that be the reason you throw the `ValidationError` and this would be a little shorter and also prevent the case that it passes the first check but throws when you cast to int?